templates/pages/comment_edits_history.html

```diff
@@ -19,27 +20,46 @@
     <li>{% trans "Historique des éditions" %}</li>
 {% endblock %}
 
-
-
 {% block headline %}
     <h1>{% trans "Historique des éditions du message" %}</h1>
 {% endblock %}
 
-
-
 {% block content_page %}
     {% if edits %}
         <table class="fullwidth">
             <thead>
+                <th></th>
                 <th class="wide">{% trans "Date" %}</th>
                 <th>{% trans "Éditeur" %}</th>
-                <th>{% trans "Version avant édition" %}</th>
+                <th>{% trans "Action" %}</th>
             </thead>
             <tbody>
+                {% sibling edits 0 as current %}
+                <tr>
+                    <td>
+                        Edition {{ edits|length }}
+                    </td>
+                    <td class="wide">{{ current.date|format_date|capfirst }}</td>
+                    <td>{% include 'misc/member_item.part.html' with member=current.editor avatar=True %}</td>
+                    <td>(Version actuelle)</td>
+                </tr>
                 {% for edit in edits %}
+                    {% sibling edits next as nextEdit %}
                     <tr>
-                        <td class="wide">{{ edit.date|format_date|capfirst }}</td>
-                        <td>{% include 'misc/member_item.part.html' with member=edit.editor avatar=True %}</td>
+                        <td>
+                            {% if forloop.revcounter0 == 0 %}
+                                Etat original
+                            {% else %}
+                                Edition {{ forloop.revcounter0 }}
```

Comment:
```suggestion
                                Édition {{ forloop.revcounter0 }}
```

Accent sur la majuscule (`Édition`).